crates/ruff/src/registry/rule_set.rs

```diff
@@ -7,10 +7,10 @@ use std::iter::FusedIterator;
 ///
 /// Uses a bitset where a bit of one signals that the Rule with that [u16] is in this set.
 #[derive(Clone, Default, CacheKey, PartialEq, Eq)]
-pub struct RuleSet([u64; 9]);
+pub struct RuleSet([u64; 10]);
 
 impl RuleSet {
-    const EMPTY: [u64; 9] = [0; 9];
+    const EMPTY: [u64; 10] = [0; 10];
```

Comment:
@arya-k - Here's the change I was referring to (very unclear from the trace, but I saw the out-of-bounds with index 9 and thought of the 9 here).

Reply:
Thank you! This is helpful, and I appreciate the fix. 

I wonder why this didn’t come up locally? I’ll do some investigating later.